DisguiseUnityRenderStream/Runtime/PipelineAbstraction/DepthCopy.cs

```diff
@@ -11,11 +10,11 @@ namespace Disguise.RenderStream
     class DepthCopy
     {
 #if UNITY_PIPELINE_HDRP && HDRP_VERSION_SUPPORTED
-        const string k_ShaderName = "Hidden/Disguise/RenderStream/DepthCopyHDRP";
+        public const string k_ShaderName = "Hidden/Disguise/RenderStream/DepthCopyHDRP";
```

Comment:
Brainstorm: I think I remember hearing somewhere they are trying to add support for both HDRP and URP in the same project, should / could we prepare for it?